lang/golang/golang/Makefile

```diff
@@ -7,8 +7,8 @@
 
 include $(TOPDIR)/rules.mk
 
-GO_VERSION_MAJOR_MINOR:=1.18
-GO_VERSION_PATCH:=8
+GO_VERSION_MAJOR_MINOR:=1.19
+GO_VERSION_PATCH:=2
```

Comment:
fwiw, .3 is already out so maybe if testing has not been done yet it makes sense to target .3 instead

Reply:
Done!